openedx2zim/scraper.py

```diff
@@ -139,6 +141,8 @@ def __init__(
         self.ignore_missing_xblocks = ignore_missing_xblocks
         self.autoplay = autoplay
         self.remove_seq_nav = remove_seq_nav
+        self.threads = threads
+        self.yt_downloader = YoutubeDownloader(threads=1 if threads == 1 else 2)
```

Comment:
We should use a single thread for youtube

Reply:
Done